cmd/agent/gui/systray/Sources/gui.swift

```diff
@@ -225,11 +238,11 @@ class AgentManager {
         let currentLoginStatus = getLoginStatus()
         var command: String
         if currentLoginStatus { // enabled -> disable
-            command = "delete every login item whose name is \"Datadog Agent\""
+            command = "/bin/launchctl unload -w " + userAgentPlistPath
         } else { // disabled -> enable
-            command = "make login item at end with properties {path:\"/Applications/Datadog Agent.app\", name:\"Datadog Agent\", hidden:false}"
+            command = "/bin/launchctl load -w " + userAgentPlistPath
```

Comment:
```suggestion
            // `load` starts a service running for the current user session, the `-w` flag will enable it going forward
            command = "/bin/launchctl load -w " + userAgentPlistPath
```

suggestion: same as above - clarifying comment on the specifics of launchctl and the -w flag